db/compaction_iterator.cc

```diff
@@ -1,5 +1,3 @@
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file. See the AUTHORS file for names of contributors.
```

Comment:
These lines are coming from levelDB isn't it? Shouldn't we let these stay (here and a few other files below)?